test/functional-tests/test.sh

```diff
@@ -188,6 +188,27 @@ function progress_bar {
     keypress
 }
 
+function icon_position {
+    for position in left top right off; do
+        tmp_dunstrc dunstrc.icon_position "icon_position = $position"
+        start_dunst dunstrc.tmp
+        for urgency in l n c; do
+            ../../dunstify -a "dunst tester" -I '../data/icons/valid.png' -u $urgency "icon_position = $position"
+        done
+        rm dunstrc.tmp
+        keypress
+    done
+}
+
+function hide_text {
+    start_dunst dunstrc.hide_text
+    ../../dunstify -a "dunst tester" -I '../data/icons/valid.png' -u n "text not hidden" "You should be able to read me!\nThe next notifications should not have any text."
+    ../../dunstify -a "dunst tester" -u l "text hidden" "If you can read me then hide_text is not working."
+    ../../dunstify -a "dunst tester" -I '../data/icons/valid.png' -u l "text hidden + icon" "If you can read me then hide_text is not working."
+    ../../dunstify -a "dunst tester" -h int:value:$((RANDOM%100)) -I '../data/icons/valid.png' -u l "text hidden + icon + progress bar" "If you can read me then hide_text is not working."
+    keypress
+}
+
```

Comment:
Here are the test notifications I get for reference:
<span> <img width="250" src="https://user-images.githubusercontent.com/85039141/149675165-fbd0e7fa-c28e-4d8f-a8a2-e4f7ca58bf6a.png" alt="icon_left"> <img width="250" src="https://user-images.githubusercontent.com/85039141/149675140-656e3fc7-8d2d-4cd2-8e16-c60eb89616a0.png" alt="icon_top"> <img width="250" src="https://user-images.githubusercontent.com/85039141/149675139-c803ad3d-332b-4b43-94e2-7eba8cebbfaf.png" alt="icon_right"> <img width="250" src="https://user-images.githubusercontent.com/85039141/149675138-47427ae7-ec3c-4f77-b3d2-de1b5581545d.png" alt="icon_off"> <img width="250" src="https://user-images.githubusercontent.com/85039141/149675135-d303d559-cf91-4541-a682-0ab17a194218.png" alt="hide_test"> </span>

Reply:
Looks good. I see see that you're using the icons in the data directory. That icon is just a small text file, so it looks like corrupted data. I would use the default icon instead by leaving out the `-I` argument. It isn't a big problem that the icon is not located inside the dunst repository.